main.ts

```diff
@@ -36,6 +38,28 @@ export default class NLSyntaxHighlightPlugin extends Plugin {
 		await this.saveData(this.settings);
 	}
 
+	async addCommands() {
+		this.addCommand({
+			id: "toggle-enabled",
+			name: "Toggle natural language highlighting",
```

Comment:
Obsidian automatically prepends the plugin's name in front of commands, so I suggest we shorten this to just "Toggle on/off"